ts/Core/Axis/AxisDefaults.ts

```diff
@@ -1028,6 +1028,7 @@ namespace AxisDefaults {
              * The pixel padding for axis labels, to ensure white space between
              * them. Defaults to 4 for horizontal axes, 1 for vertical.
              *
+             * @type      {number|undefined}
```

Comment:
`undefined` is an implicit type because it is an user option, so it can be skipped.

Reply:
Changed. Thanks!